libs/exponential-histogram/src/test/java/org/elasticsearch/exponentialhistogram/ExponentialHistogramBuilderTests.java

```diff
@@ -68,18 +73,6 @@ public void testBuildWithEstimation() {
         }
     }
 
-    public void testAddDuplicatePositiveBucketThrows() {
```

Comment:
Maybe keep those tests and verify that the vales are overridden in the `ReleasableExponentialHistogram`.